src/content/docs/pt-br/reference/configuration-reference.mdx

```diff
@@ -712,29 +793,22 @@ Você pode usar coringas para definir os valores permitidos de `hostname` e `pat
   - Termine com '/**' para permitir todas as sub-rotas ('começa com').
   - Termine com '/*' para permitir apenas um nível de sub-rota.
 
+## Opções da Barra de ferramentas de Desenvolvimento
 
-## Opções de Markdown
 
-### markdown.drafts
+### devToolbar.enabled
 
 <p>
 
-**Tipo:** `boolean`<br />
-**Padrão:** `false`
+**tipo:** `boolean`<br />
+**padrão:** `true`
```

Comment:
```suggestion
**Tipo:** `boolean`<br />
**Padrão:** `true`
```
These seem to be capitalized everywhere else!

Reply:
I think I kept that inconsistency from the original text 😅